DEPLOY.md

```diff
@@ -0,0 +1,67 @@
+# How to deploy the Go Fast contracts
+
+## Deploying the Cosmwasm contract
+
+### 1. Create a `.env` file in the `cosmwasm` directory with the following:
+
+```
+DEPLOYER_MNEMONIC=<your_mnemonic>
+```
+
+### 2. Store the contract:
+
+```
+npx ts-node scripts/storeContract.ts
```

Comment:
I thought it wasn't possible to do this on osmosis without being whitelisted? or are we just ignoring that and assuming they will just deploy on a different cosmwasm chain for testing